db/pre-re/item_db.conf

```diff
@@ -22075,8 +22075,8 @@ item_db: (
 	Loc: 256
 	View: 61
 	Script: <"
-		bonus bAgi,2;
-		bonus2 bSubEle,Ele_Ghost,10;
+		bonus bAgi, 2;
+		bonus2 bSubEle, Ele_Ghost, 10;
```

Comment:
pre-renewal kRO is hard to find this day,
I'll use twRO's pre-renewal instead.
And yes, it's 15%
ftp://ropatch.gameflier.com/Patch/2010-05-17itemdata_main.gpf